contracts/core-token-vesting-v2/src/contract.rs

```diff
@@ -77,13 +77,8 @@ pub fn execute(
         ExecuteMsg::DeregisterVestingAccounts { addresses } => {
             deregister_vesting_accounts(deps, env, info, addresses)
         }
-        ExecuteMsg::Claim {
-            denoms: _denoms,
-            recipient,
-        } => claim(deps, env, info, recipient),
-        ExecuteMsg::Withdraw { amount, recipient } => {
-            withdraw(deps, env, info, amount, recipient)
-        }
+        ExecuteMsg::Claim { denoms: _denoms } => claim(deps, env, info),
```

Comment:
If we do breaking changes for the interface we can get rid of the denom parameter also

Reply:
we can leave it then